source/server/admin/admin.h

```diff
@@ -270,6 +270,9 @@ class AdminImpl : public Admin,
     struct NullThreadLocalOverloadState : public ThreadLocalOverloadState {
       NullThreadLocalOverloadState(Event::Dispatcher& dispatcher) : dispatcher_(dispatcher) {}
       const OverloadActionState& getState(const std::string&) override { return inactive_; }
+      bool tryAllocateResource(OverloadProactiveResourceName, int64_t) override { return false; }
+      bool tryDeallocateResource(OverloadProactiveResourceName, int64_t) override { return false; }
+      int64_t currentResourceUsage(OverloadProactiveResourceName) override { return 0; }
```

Comment:
this was introduced for testing purposes, may need to move it from interface to impl class.